nichtparasoup.py

```diff
@@ -218,21 +220,12 @@ def cache_status():
 
                 count = info["images_per_site"][key]
 
-                # "Drawing" a Bar to visualize the cache status
-                sharps_percent = min_cache_imgs_before_refill * 100 / min_cache_imgs
-                count_percent = count * 100 / min_cache_imgs
-
-                bar = "["
-                for i in range(1,15):
-                    if i * 10 < count_percent:
-                        if i * 10 < sharps_percent:
-                            bar += "#"
-                        else:
-                            bar += "*"
-                    elif i <= 10:
-                        bar += "_"
-                    if i == 10:
-                        bar += "]"
+                bar = "|"
+                for i in range(0, count / 5):
+                    if i < min_cache_imgs_before_refill / 5:
+                        bar += "#"
+                    else:
+                        bar += "*"
```

Comment:
Since there is no "upper border" in the Cache, Bars are now looking like this:
Pr0gramm - new             with factor  1.0: 36 Images |###****
One Char in the bar is representing 5 images. In this example we have 15 images as lower border represented as sharps and 21 images above the lower border as stars ( four times full 5 images, so 5 stars )